manifests/server.pp

```diff
@@ -77,7 +77,7 @@
 # @param roles Specifies a hash from which to generate postgresql::server::role resources.
 # @param config_entries Specifies a hash from which to generate postgresql::server::config_entry resources.
 # @param pg_hba_rules Specifies a hash from which to generate postgresql::server::pg_hba_rule resources.
-
+#
```

Comment:
Interestingly enough, this is why we have https://github.com/voxpupuli/puppet-lint-param-docs. However, it's disabled here and somehow doesn't catch it. That's a bug.

Reply:
Amazing @ekohl thanks for this find!

Reply:
It's the most common mistake I've seen with puppet-strings. When things disappear, look for empty lines.